#!/usr/bin/env python
import math

PACKAGE = "egocylindrical"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("num_threads", int_t, 0, "# threads for generating range image", 1, 1, 16)

exit(gen.generate(PACKAGE, "egocylindrical", "RangeImageGenerator"))
